src/components/Text.tsx

```diff
@@ -7,14 +7,20 @@ interface TextProps {
   style?: TextStyle
   testID?: string
   onPress?: () => void
+  allowFontScaling?: boolean
```

Comment:
Im a bit wary of this pattern fo passing Text props here, i wonder can find a better way to apply writing direction style.  Wouldnt change anything in this pr though.